Sources/MintKit/Mint.swift

```diff
@@ -99,12 +100,11 @@ public struct Mint {
         try install(package, update: false, verbose: verbose, global: false)
         print("🌱  Running \(package.commandVersion)...")
 
-        var context = CustomContext(main)
-        context.env["MINT"] = "YES"
-        context.env["RESOURCE_PATH"] = ""
-
         let packagePath = PackagePath(path: packagesPath, package: package)
-        try context.runAndPrint(packagePath.commandPath.string, arguments)
+        try shell.runAndPrint(packagePath.commandPath.string, arguments, env: [
+            "MINT": "YES",
+            "RESOURCE_PATH": ""
```

Comment:
I guess we can move these env variables into the default implementation, but in different PR.